src/controllers/base.controller.js

```diff
@@ -27,12 +23,15 @@ module.exports = class BaseController {
     return pageContext
   }
 
-  static handler (request, reply, errors, controllerSubclass, validateToken = true) {
-    if (validateToken) {
-      // Validate the session cookie
-      let token = request.server.methods.validateToken(request.state[Constants.COOKIE_KEY])
-      if (!token) {
+  static handler (request, reply, errors, controllerSubclass, cookieValidationRequired = true) {
+    if (cookieValidationRequired) {
+      const cookie = request.state[Constants.COOKIE_KEY]
+
+      // Validate the cookie
+      if (!CookieService.validateCookie(cookie)) {
```

Comment:
We have found this morning that if the cookie doesn't exist i.e. you go straight to a route without having first created a cookie, you don't get redirected to the error page and instead get a 500.

Its because `validateCookie()` in the `CookieService` throws an error if the cookie doesn't exist, but this line is expecting a simple true or false.

Therefore we either need a separate check and method for if the cookie exists, or `validateCookie()` needs to become `cookieExistsAndIsValid()` and that method only returns a true or false no matter if there is an exception.